fennel/client_tests/test_social_network.py

```diff
@@ -62,7 +61,7 @@ class CityInfo:
     @inputs(UserInfo)
     def count_city_gender(cls, user_info: Dataset):
         return user_info.groupby(["city", "gender"]).aggregate(
-            [Count(window=Window("1y 5s"), into_field="count")]
+            [Count(window=Window("6y 8s"), into_field="count")]
```

Comment:
I believe we have removed the requirement to specify Window, so you can directly specify "6y 8s"

Reply:
will change this.